isis/src/qisis/objs/CnetEditorWidget/CnetEditorSortConfigDialog.cpp

```diff
@@ -23,6 +23,8 @@ namespace Isis {
     CnetEditorWidget *cnetWidget) : QDialog(cnetWidget) {
     m_cnetWidget = cnetWidget;
 
+    setWindowTitle("Table Sorting Dialog");
```

Comment:
Do we want this to be called Table Sorting Dialog? Why not Table Sorting?

Reply:
I was not sure what to name it. Tracie said it was okay for now, but I can change it to whatever is the best.

Reply:
I personally don't like Dialog at the end

Reply:
This is sort of a gui standard (appending the Dialog to the class name).  TableSortingDialog indicates this is a gui dialog class.  To me, TableSorting indicates that the class actually performs the table sorting.

Reply:
The window title is the title of the dialog that pops up in ipce, not the class itself.

Reply:
OOps, sorry thought you were talking about the class itself.  Yes, I definitely agree with Makayla the title should not have the word Dialog.  Sorry for my confusion.